### PR TITLE
Lista2

### PR DESCRIPTION
**Pull request by wiktorkowalski**
zad 7 & zad 3

### CODE REVIEW

File: lista2/zadanie3/Form1.cs [hunk 1]
```diff
@@ -10,37 +10,18 @@ public Form1()
...
+            this.comboBox3.DataSource = Enum.GetValues(typeof(MessageBoxDefaultButton));
+            this.comboBox4.DataSource = Enum.GetValues(typeof(MessageBoxOptions));
 
         }
 
-        private void button6_Click(object sender, EventArgs e)
+        private void button1_Click(object sender, EventArgs e)
         {
-
+            this.labelResult.Text = MessageBox.Show("content", "title", (MessageBoxButtons)this.comboBox1.SelectedItem, (MessageBoxIcon)this.comboBox2.SelectedItem, (MessageBoxDefaultButton)this.comboBox3.SelectedItem, (MessageBoxOptions)this.comboBox4.SelectedItem).ToString();
```
Comment (mikolajkalwa) [inline lista2/zadanie3/Form1.cs]: this line is too long

File: lista2/zadanie7/Codec.cs [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            Flags = item.Flags;
+        }
+
+        public string Name { get; set; }
+        public string Extension { get; set; }
+        public ImageCodecFlags Flags { get; set; }
+        public override string ToString()
+        {
+            return Name;
+            //return $"Name: {Name}\nExtension type: {Extension}\nFlags: {Flags}";
```
Comment (mikolajkalwa) [inline lista2/zadanie7/Codec.cs]: commented code

**Review by mikolajkalwa — CHANGES_REQUESTED** [review]

File: lista2/zadanie3/Form1.cs (hunk 1 above)
Comment (wiktorkowalski) [reply lista2/zadanie3/Form1.cs]: then buy a bigger monitor

File: lista2/zadanie7/Codec.cs (hunk 2 above)
Comment (wiktorkowalski) [reply lista2/zadanie7/Codec.cs]: its important

**Review by mikolajkalwa — APPROVED** [review]

**Pull request merged**